tests/CMakeLists.txt

```diff
@@ -104,7 +104,7 @@ endforeach()
 add_subdirectory(psi4numpy)
 add_subdirectory(python)
 add_subdirectory(json)
-add_subdirectory(pasture)
+add_subdirectory(pasture-ccsorttransqt2)
```

Comment:
Any reason not to add a `if(ENABLE_ccsort OR ENABLE_transqt2)` around this and the "Adding Psi4 + transqt2 tests" message?